### PR TITLE
In which we add Linkfire to MDFP

### LINKED ISSUE #2288: MDFP Issue: Static Content domains are being blocked linkstorage & st .linkfire.com
#### What is your browser and browser version?
Chrome Version 71.0.3578.98
#### What is broken and where?
When visiting lnk.to domains, our static content domains are being blocked.  linkfire.com

#### What is the "culprit" domain?
I'd like to open this issue as a MDFP issue, as I can tell its our first party domain that is getting blocked.  

The following are the 2 culprit domains
linkstorage.linkfire.com
st.linkfire.com

#### What is your debug output for this domain?
To get the debug output, please see the instructions link above.
```
**** ACTION_MAP for linkfire.com
authentication-beta-internal-api.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1540168498572,
  "userAction": ""
}
authentication-internal-api.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1548111122929,
  "userAction": ""
}
authentication-test-internal-api.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1540653290737,
  "userAction": ""
}
build.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1541031264780,
  "userAction": ""
}
druid-test-broker.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1541645220712,
  "userAction": ""
}
frontpagegui.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1549020940027,
  "userAction": ""
}
guibeta.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1540864945281,
  "userAction": ""
}
guiproduction.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1540642552036,
  "userAction": ""
}
guitest.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1540922134163,
  "userAction": ""
}
help.linkfire.com {
  "dnt": false,
  "heuristicAction": "allow",
  "nextUpdateTime": 1545233682181,
  "userAction": ""
}
linkfire.com {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1530134460761,
  "userAction": ""
}
linkstorage.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1549621584967,
  "userAction": ""
}
services.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1532022582670,
  "userAction": ""
}
st.linkfire.com {
  "dnt": false,
  "heuristicAction": "block",
  "nextUpdateTime": 1549329542518,
  "userAction": ""
}
styleguide.linkfire.com {
  "dnt": false,
  "heuristicAction": "",
  "nextUpdateTime": 1542022015280,
  "userAction": "user_allow"
}
**** SNITCH_MAP for linkfire.com
linkfire.com [
  "lnk.to",
  "zendesk.com",
  "strm.to"
]
```

### PR DESCRIPTION
**Pull request by adyrcz**
This pull request fixes #2288.

### CODE REVIEW

**Comment by adyrcz** [comment]
Not sure why it says All checks have failed
The build only failed on the test: tests/selenium/dnt_test.py::DNTTest::test_dnt_check_should_not_set_cookies


https://travis-ci.org/EFForg/privacybadger/jobs/487928214

**Comment by ghostwords** [comment]
Hello! The failing test isn't related to your PR.

File: src/js/multiDomainFirstParties.js [hunk 1]
```diff
@@ -1318,7 +1319,6 @@ var multiDomainFirstPartiesArray = [
     "springerprofessional.de",
   ],
   ["sprint.com", "sprintpcs.com", "nextel.com"],
-  ["squareup.com", "cash.app", "mkt.com", "squarecdn.com"],
```
Comment (ghostwords) [inline src/js/multiDomainFirstParties.js]: This line should be getting removed.

Comment (adyrcz) [reply src/js/multiDomainFirstParties.js]: oh boy!  Sorry Square!!!  Fixing that now

**Pull request merged**